droidlet/interpreter/craftassist/mc_stop_condition.py

```diff
@@ -4,15 +4,15 @@
 import os
 import sys
 
-from droidlet.interpreter.stop_condition import StopCondition
+from droidlet.interpreter.condition import Condition
 
-
-class AgentAdjacentStopCondition(StopCondition):
+# FIXME!!!! this is dead code
```

Comment:
lets kill it rightaway!

Reply:
fixed in c576a26cca76c32b5ce4fdfd9d89faec68c9d96b